packages/components/src/template-registry.ts

```diff
@@ -187,11 +187,16 @@ import type HdsTagComponent from './components/hds/tag';
 import type HdsTooltipButtonComponent from './components/hds/tooltip-button';
 import type HdsToastComponent from './components/hds/toast';
 import type HdsTextCodeComponent from './components/hds/text/code';
+import type HdsTimeComponent from './components/hds/time';
+import type HdsTimeSingleComponent from './components/hds/time';
+import type HdsTimeRangeComponent from './components/hds/time';
```

Comment:
```suggestion
import type HdsTimeSingleComponent from './components/hds/time/single';
import type HdsTimeRangeComponent from './components/hds/time/range';
```

Reply:
suggestion: this change should address most of the remaining [Type issues you mentioned](https://github.com/hashicorp/design-system/pull/2515#pullrequestreview-2422417434)

Reply:
thank you!